Blish HUD/Controls/TextInputBase.cs

```diff
@@ -187,8 +187,12 @@ public abstract class TextInputBase : Control {
         /// </summary>
         public int Length => _text.Length;
 
+        /// <summary>
+        /// Get state of modifier keys
+        /// </summary>
```

Comment:
Just FYI, this will only apply this summary to the `IsShiftDown`.  It won't apply it to the two below it.